CONTRIBUTORS.md

```diff
@@ -24,5 +24,6 @@
 |Lakshya Mutneja     | [lakshyamutneja](https://github.com/lakshyamutneja) | Cloud Computing, Python |
 |                    |                                                 |                            |
 |Ayan Raj            | [ayanujju](https://github.com/ayanujju)         | Machine Learning, Web Dev  |
-|Aaadish Kala        | [Aadish09](https://github.com/Aadish09)         | Web Dev ,UI/UX             |
```

Comment:
Please add this line back

Reply:
Sure, sorry for the inconvenience caused.